### ISSUE TITLE
Zero outstanding payment communication after ACTUALs with Updated Receipt template

### ISSUE THREAD

**Issue by abin-aot** [bcgov/foi-flow#3016]
As of now, we are **NOT** sending an email to the applicant , if "**ACTUAL**"  tallies outstanding amount to zero, if applicant has paid half deposit on the total amount. See below example

![image.png](https://images.zenhubusercontent.com/607de5aafedfab08839427c4/064cc55c-9349-4d32-93d8-3f69c232bf0d)

* As an applicant who paid a deposit(half) on a Fee estimate (Suppose applicant paid 315 as deposit on a 630 full estimate)
*  Then, if Ministry and IAO enters **actuals** to make/tallies outstanding to **0.00** then applicant should receive an  email message with '**UPDATED**' receipt with actuals on it - with all tabulations 
* so that there is NO confusion with applicant , how the calculation went, what was the actuals. 

**Assumptions & Scope**	
What are the assumptions for this story?

What is IN scope?

What is NOT in scope?

**Acceptance Criteria**	

**Scenario 1: xxxxxx**
* GIVEN ... (you or your condition)
* WHEN ... (what you do) 
* THEN ... (what you see) 

**Scenario 2: xxxxxx**
* GIVEN ... (you or your condition)
* WHEN ... (what you do) 
* THEN ... (what you see) 

**Scenario 3: xxxxxx**
...

**Dependencies? What is the impact of this dependency? (If so, link dependency in the ticket, make it visible in a team´s backlog)**

**Validation Rules? (If yes, list here)**	

**Design**
@xxx - please link the Design here

**Definition of Ready**

1. [ ] Is there a well articulated User Story?
2. [ ] Is there Acceptance Criteria that covers all scenarios (happy/sad paths)?
3. [ ] If there is a user interface, is there a design? 
4. [ ] Does the user story need user research/validation?
5. [ ] Does this User Story needs stakeholder approval?
6. [ ] Design / Solution accepted by Product Owner
7. [ ] Is this user story small enough to be completed in a Sprint? Should it be split?
8. [ ] Are the dependencies known/ understood? (technical, business, regulatory/policy)
9. [ ] Has the story been estimated?

**Definition of Done**

1.    [ ] Passes developer unit tests
2.    [ ] Passes peer code review
3.    [ ] If there's a user interface, passes UX assurance
4.    [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
5.	[ ] Confirm Test cases built and succeeding 
6.	[ ] No regression test failures
7.	[ ] Test coverage acceptable by Product Owner
8.	[ ] Ticket ready to be merged to master or story branch
9.	[ ] Developer to list Config changes/ Update documents and designs
10.	[ ] Can be demoed in Sprint Review 
11.	[ ] Tagged as part of a Release
12.   [ ] Feature flagged if required
13.  [ ] Change Management activities done?

**Comment by lmullane** [comment]
This would be part of the refund flow.

If a refund is due than the analyst or the system should notify the Intake Manager that a refund is due. The Intake Manager would then need to approve and send the request by email to process the refund.

Only after that has happened should the applicant be notified of the refund. 

The next step is to confirm the flow with the Intake Manager.
